SimPEG/electromagnetics/viscous_remanent_magnetization/simulation.py

```diff
@@ -934,10 +934,10 @@ def Jvec(self, m, v, f=None):
         dxidm = self.xiMap.deriv(m)
 
         # dxidm*v
-        v = np.matrix(dxidm * v).T
+        v = np.atleast_2d(dxidm * v).T
```

Comment:
this shouldn't need to be here at all either
`v = dxidm * v`
(if I'm keeping track of dimensions correctly)